types/oidc-provider/index.d.ts

```diff
@@ -0,0 +1,2014 @@
+// Type definitions for oidc-provider 7.1
+// Project: https://github.com/panva/node-oidc-provider
+// Definitions by: Filip Skokan <https://github.com/panva>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference lib="dom"/>
```

Comment:
This looks like a node library; what is this for?

Reply:
JWK definition.

Reply:
The problem is that this will cause anyone who uses this library to pull in _all_ of lib.dom.d.ts, which most Node library authors don’t want. This is actually a case where it would be better to just copy that interface.

Reply:
ok, i'll do so.

Reply:
done.